pubspec.yaml

```diff
@@ -24,8 +24,8 @@ environment:
   # that by the time we want to release, these will have become stable.
   # TODO: Before general release, switch to stable Flutter and Dart versions,
   #   or pin exact versions: https://github.com/zulip/zulip-flutter/issues/15
-  sdk: '>=3.4.0-256.0.dev <4.0.0'
-  flutter: '>=3.21.0-12.0.pre.26'
+  sdk: '>=3.4.0-140.0.dev <4.0.0'
```

Comment:
Never change the sdk version in pubspec.yaml as it is configured for the whole team. refer to [ReadMe](https://github.com/zulip/zulip-flutter?tab=readme-ov-file#flutter-version)